### DISCUSSION TITLE
What permissions are needed for the PAT?

### DISCUSSION THREAD

**Discussion by rtanzolaep** [some-natalie/ghas-to-csv#41] [Q&A]
What permissions are needed for a fine-grained PAT at the various levels to support this action running at the repo, org, or enterprise level?

**Comment by some-natalie** [comment]
Hey @rtanzolaep , I didn't test against fine-grained PATs and I'm not sure the API permissions are the same there.  Did you try with a regular PAT?

**Reply by rtanzolaep** [reply]
Thank you, @some-natalie, I'll give it a try with a regular PAT.
